client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -4,7 +4,7 @@ import {
 } from '@automattic/calypso-products';
 import { withShoppingCart } from '@automattic/shopping-cart';
 import { useTranslate } from 'i18n-calypso';
-import React, { FunctionComponent } from 'react';
+import React, { FunctionComponent, useState } from 'react';
```

Comment:
Nit: I think we can import `FunctionComponent` as a type.
```suggestion
import React, { useState } from 'react';
import type { FunctionComponent } from 'react';
```